qoffeeapi/qoffeeapi/hc_connector.py

```diff
@@ -75,8 +75,8 @@ def get_connector():
     api_base_url = os.getenv("HOMECONNECT_API_URL")
     _HOMECONNECT_CONNECTOR = HomeconnectConnector(
         api_base_url=api_base_url,
-        authorize_url=api_base_url+"/security/oauth/authorize",
-        token_url=api_base_url+"/security/oauth/token",
+        authorize_url=joinUrl(api_base_url, "/security/oauth/authorize"),
```

Comment:
Very good idea, actually there is already a package in standard library for this (`urllib.parse`).

Reply:
Habs jetzt angepasst wie besprochen